src/bitaxe_monitor.py

```
@@ -2154,6 +2154,28 @@
                 })
             except Exception as e:
                 return jsonify({'error': str(e)}), 500
+        
+        @self.app.route('/debug')
+        def debug_page():
+            """Serve debug page to test API endpoints"""
+            try:
+                with open('debug_web.html', 'r', encoding='utf-8') as f:
+                    return f.read()
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)